### DISCUSSION TITLE
Simulateur location meublée

### DISCUSSION THREAD

**Discussion by mquandalle** [betagouv/mon-entreprise#2616] [Nouveaux simulateurs]
Je suis en train de lire https://boutique.grouperf.com/produit/les-locations-en-meuble-2022, qui est exhaustif et très clair, et il me semble qu'il y a certains cas non pris en compte dans le simulateur meublé du parcours économie collaborative.

Par ailleurs il y aurait un intérêt à extraire ce sous-simulateur sur une page indépendant référencée « simulateur de location en meublé ».

Tags : LMP ; LMNP

**Comment by liliced** [comment]
Comparateur traité dans https://github.com/betagouv/mon-entreprise/issues/3371
Calcul des cotisations en attente de https://github.com/betagouv/mon-entreprise/issues/2915
